llvm/test/ThinLTO/AMDGPU/force-import-all.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: opt -mtriple=amdgcn-amd-amdhsa -module-summary %s -o %t.main.bc
+; RUN: opt -mtriple=amdgcn-amd-amdhsa -module-summary %p/Inputs/in-0.ll -o %t.in.0.bc
+; RUN: opt -mtriple=amdgcn-amd-amdhsa -module-summary %p/Inputs/in-1.ll -o %t.in.1.bc
+; RUN: llvm-lto -thinlto-action=run -force-import-all %t.main.bc %t.in.0.bc %t.in.1.bc --thinlto-save-temps=%t.2.
```

Comment:
Don't you just need to run `-thinlto-action=import` instead of running the full pipeline and rely on save-temps?

Reply:
I wanted to do that as well, but it looks like `thinLTOResolvePrevailingGUID` doesn't run if we simply does `import`, as shown below:

```
; RUN: llvm-lto -thinlto-action=thinlink -force-import-all %t.main.bc %t.in.bc -o %t.index.bc
; RUN: llvm-lto -thinlto-action=import -force-import-all %t.main.bc %t.in.bc --thinlto-index=%t.index.bc
; RUN: llvm-dis %t.main.bc.thinlto.imported.bc -o - | FileCheck --check-prefix=MOD1 %s
; RUN: llvm-dis %t.in.bc.thinlto.imported.bc -o - | FileCheck --check-prefix=MOD2 %s
```

Did I miss anything here?